c++/src/capnp/any-test.c++

```diff
@@ -114,6 +114,12 @@ TEST(Any, AnyStruct) {
   checkTestMessage(root.getAnyPointerField().getAs<TestAllTypes>());
   checkTestMessage(root.asReader().getAnyPointerField().getAs<TestAllTypes>());
 
+  auto all_types = root.getAnyPointerField().getAs<AnyStruct>().as<TestAllTypes>();
```

Comment:
Unwritten KJ style rule: Use camel-case, not underscores: `allTypes`, `allTypesReader`.